docs/source/data.rst

```diff
@@ -34,3 +34,9 @@ Eye
 
 .. automodule:: aopy.data.eye
     :members:
+
+Neuropixel
+^^^
+
+.. automodule:: aopy.data.Neuropixel
```

Comment:
not sure if it matters, but `aopy.data.neuropixel` and `aopy.preproc.neuropixel` aren't capitalized in the filename